GameOfLife/GameOfLife/Engine.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GameOfLife;
+
+namespace GameOfLife
+{
+    public class Engine
+    {
+        int height;
+        int width;
+        int generation = 1;
+        string[,] gameField;
+
+        /// <summary>
+        /// Initiate field size choice
+        /// </summary>
+        public void Start()
+        {       
+            string fieldSizeChoice;
+
```

Comment:
In general empty line is needed after code block followed by code line, but not in case of closing brace.
This and other empty lines should be removed.

Reply:
Removed redundant empty lines and took into consideration for the future.